### PR TITLE
Added easy.js tests with Jest 

### PR DESCRIPTION
**Pull request by AleksArnau**
- easy.js was cleaned up from comments and imported to easy.test.js with Jest for testing
- package.json was updated with Babel to run ESM
- .gitignore was updated to include build as a result of using Babel
- all the X.test.js had a placeholder test added to them

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-node_modules
+build
+node_modules
```
Comment (peterthehan) [inline .gitignore]: add a new line at eof

File: package.json [hunk 2]
```diff
@@ -1,13 +1,15 @@
 {
-  "name": "JSExercises",
+  "name": "js-exercises",
   "version": "1.0.0",
   "description": "I use [VS Code](https://code.visualstudio.com/) to edit code, [Prettier](https://prettier.io/) for formatting",
-  "main": "index.js",
+  "main": "build/index.js",
```
Comment (peterthehan) [inline package.json]: i don't think you have a index.js entry point, lets remove this kv pair from the file

File: package.json [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
   "directories": {
     "test": "test"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest",
+    "prebuild": "rm -rf build/",
+    "build": "npx babel src -d build"
```
Comment (peterthehan) [inline package.json]: nit: sort the keys alphabetically.

**Review by peterthehan — COMMENTED** [review]
did a first pass, looks good once resolved

File: package.json (hunk 2 above)
Comment (AleksArnau) [reply package.json]: what's a kv pair

Comment (peterthehan) [reply package.json]: key-value

**Review by peterthehan — APPROVED** [review]
lgtm

**Pull request merged**